http/oidc/src/main/java/org/wildfly/security/http/oidc/TokenValidator.java

```diff
@@ -288,6 +297,38 @@ private static String getAccessTokenHash(String accessTokenString, String jwsAlg
 
     }
 
+
+    private static class NonceValidator implements ErrorCodeValidator {
+        private final String stateCookieValue;
```

Comment:
Maybe sessionRandomCookieValue might be better to use to avoid confusion with the state cookie.

Reply:
done

Reply:
changed per previous comments